android/src/main/cpp/graphics/OpenGlContext.h

```diff
@@ -46,11 +46,17 @@ class OpenGlContext : public RenderingContextInterface, public std::enable_share
 
     virtual float getAspectRatio();
 
+    virtual long getDeltaTimeMs();
+
   protected:
     RenderingCullMode cullMode = RenderingCullMode::NONE;
+    std::atomic_flag backgroundColorValid = ATOMIC_FLAG_INIT;
     Color backgroundColor = Color(0, 0, 0, 1);
 
     std::unordered_map<std::string, int> programs;
 
     Vec2I viewportSize = Vec2I(0, 0);
+
+    std::chrono::milliseconds timeCreation;
```

Comment:
missing include for `<chrono>` https://github.com/openmobilemaps/maps-core/actions/runs/14086789718/job/39452876080#step:6:340